doc/user-guide/data_model.rst

```diff
@@ -696,21 +701,17 @@ these vectors represent quantities with direction.
    VTK permits only one "active" vector.  If you have multiple vector
    arrays that you wish to transform, set
    ``transform_all_input_vectors=True`` in :func:`transform()
-   <pyvista.core.dataset.DataSet.transform>`.  Be aware that this will
+   <pyvista.DataSet.transform>`.  Be aware that this will
    transform any array with three components, so multi-component
    scalar arrays like RGB arrays will have to be discarded after
    transformation.
 
-
 .. _vtkDataArray: https://vtk.org/doc/nightly/html/classvtkDataArray.html
 .. _vtkDataSet: https://vtk.org/doc/nightly/html/classvtkDataSet.html
 .. _vtkFieldData: https://vtk.org/doc/nightly/html/classvtkFieldData.html
 .. _vtkDataObject: https://vtk.org/doc/nightly/html/classvtkDataObject.html
-.. _vtk.vtkPolyData: https://vtk.org/doc/nightly/html/classvtkPolyData.html
-.. _vtk.UnstructuredGrid: https://vtk.org/doc/nightly/html/classvtkUnstructuredGrid.html
-.. _vtk.vtkStructuredGrid: https://vtk.org/doc/nightly/html/classvtkStructuredGrid.html
-.. _vtk.vtkRectilinearGrid: https://vtk.org/doc/nightly/html/classvtkRectilinearGrid.html
 .. _vtkImageData: https://vtk.org/doc/nightly/html/classvtkImageData.html
-.. _vtk.vtkMultiBlockDataSet: https://vtk.org/doc/nightly/html/classvtkMultiBlockDataSet.html
 
-.. _cast_to_unstructured_grid: :func:`cast_to_unstructured_grid <pyvista.core.dataset.DataSet.cast_to_unstructured_grid>`
+.. |PolyData| replace:: :class:`PolyData <pyvista.PolyData>`
+.. |UnstructuredGrid| replace:: :class:`UnstructuredGrid <pyvista.UnstructuredGrid>`
+.. |UniformGrid| replace:: :class:`UniformGrid <pyvista.UniformGrid>`
```

Comment:
I could only get these shortened intra-doc references to work using this syntax. It's a bit unwieldy, but I couldn't get anything else to work.